packages/components/Footer.tsx

```diff
@@ -30,24 +28,29 @@ function PodCodarLogo() {
 }
 
 function Pix() {
-  const { t } = useI18n("footer");
   return (
     <div className="flex flex-col items-center gap-2">
-      <p>{t("contribution")}</p>
+      <p>
+        <LocalizedText translation={"footer.contribution"} />
+      </p>
       <p className="text-sm text-zinc-500">{PIX_KEY}</p>
-      <Image src={images.pixQRCode} width={125} height={125} alt={t("contribution")} />
+      <Image src={images.pixQRCode} width={125} height={125} alt={"qr code"} />
     </div>
   );
 }
 
 function Copyrights() {
-  const { t } = useI18n("footer");
   const currentYear = new Date().getFullYear();
   return (
     <div className="flex flex-col items-center gap-2">
-      <p>{t("podcodar")}</p>
+      <p>
+        <LocalizedText translation={"footer.podcodar"} />
+      </p>
       <SocialIconLinks />
-      <p className="text-sm">{t("legal", { currentYear })}</p>
+      <p className="text-sm">
+        <LocalizedText translation="footer.legal" />
+        {currentYear}
```

Comment:
While testing the preview version (https://webapp-git-marco-setup-localized-text-component-podcodar.vercel.app), the `currentYear` is not being rendered correctly. Could you please take a look and fix this?

![image](https://github.com/user-attachments/assets/441e5551-dfa6-4f54-8ff6-fa61649b7e8f)